Fixing the following warning:

attempted to register the menu item "<Save>" for procedure "python-fu-contact-sheet".

--- a/contact-sheet/contactsheet.py
+++ b/contact-sheet/contactsheet.py
@@ -594,7 +594,6 @@ register(
         ],
         [],
         Contact_Sheet,
-        menu="<Save>",
         domain=("contactsheet", gimp.locale_directory)
         )
 
